.travis.yml

```diff
@@ -86,6 +86,10 @@ jobs:
       env: PDB_TEST=core+ext/openjdk8/pg-9.6
       script: *run-core-and-ext-tests
 
+    - stage: ❧ pdb tests with FIPS
+      env: PDB_TEST=core+ext/openjdk8/pg-9.6 LEIN_PROFILES=FIPS
+      script: *run-core-and-ext-tests
+
```

Comment:
I think we have very limited usage of SSL in our core tests, this might better test our setup in integration tests (but that relies on being able to run puppetserver and Puppet in FIPs mode too. And our certs over in `test-resources/` might not be adequate for FIPs testing, so running integration tests in FIPS mode should not hold up this PR and is probably an idea for post-Lovejoy

Reply:
I'd like to keep at least one set of unit tests that run in this mode, just as an early warning system. puppetserver had unit tests that failed quickly in FIPS mode. Adding integration tests too is a great idea.

Our certs should work - they're read as PEM files and a FIPS keystore is created in memory. I can run puppetdb in FIPS mode with a pgbox set of certs.